samples/client/petstore/R/LICENSE

```diff
@@ -0,0 +1,2 @@
+YEAR: 2017
+COPYRIGHT HOLDER: Ramkumar Chandrasekaran
```

Comment:
@ramnov FYI. The auto-generated code is unlicensed so we won't generate this file.